tests/visualization_tests/test_parallel_coordinate.py

```diff
@@ -179,6 +231,60 @@ def test_plot_parallel_coordinate_log_params() -> None:
     assert figure.data[0]["dimensions"][2]["tickvals"] == (1.0, 2.0, math.log10(200))
 
 
+def test_plot_parallel_coordinate_unique_hyper_param() -> None:
+    # Test case when one unique value is suggested during the optimization.
+
+    study_categorical_params = create_study()
+    study_categorical_params.add_trial(
+        create_trial(
+            value=0.0,
+            params={"category_a": "preferred", "param_b": 30},
+            distributions={
+                "category_a": CategoricalDistribution(("preferred", "opt")),
+                "param_b": LogUniformDistribution(1, 1000),
+            },
+        )
+    )
+
+    # both hyperparameters contain unique values
```

Comment:
Sorry, one more comment on here
```suggestion
    # Both hyperparameters contain unique values.
```